src/Form/ImportVideoItemByMpxId.php

```diff
@@ -71,19 +72,22 @@ public function buildForm(array $form, FormStateInterface $form_state) {
 
     $form['video_type'] = [
       '#type' => 'select',
-      '#title' => $this->t('Video Type'),
+      '#title' => $this->t('Video type'),
       '#description' => $this->t('Choose the video type to import the video into.'),
       '#options' => $video_opts,
       '#required' => TRUE,
     ];
     $form['mpx_id'] = [
       '#type' => 'textfield',
-      '#title' => t('mpx item ID'),
+      '#title' => t('ID'),
+      '#placeholder' => 'Type the ID of the mpx video you want to import.',
```

Comment:
No `t()` ?

Reply:
Good catch, will fix.

Reply:
Can we use StringTranslationTrait too?

Reply:
Yep absolutely, it's already being used in this class.